content/en/blog/_posts/2017-09-00-Windows-Networking-At-Parity-With-Linux.md

```diff
@@ -3,6 +3,8 @@ title: " Windows Networking at Parity with Linux for Kubernetes "
 date: 2017-09-08
 slug: windows-networking-at-parity-with-linux
 url: /blog/2017/09/Windows-Networking-At-Parity-With-Linux
+author: >
+   Jason Messer (Microsoft)
 ---
 _**Editor's note: today's post is by Jason Messer, Principal PM Manager at Microsoft, on improvements to the Windows network stack to support the Kubernetes CNI model.**_  
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.